templates/index.html

```diff
@@ -310,34 +310,36 @@ <h2>The Snap Store</h2>
           <p class="p-heading--five">The Snap Store is enabled by default on tens of millions of Linux systems. It allows developers to release free or paid apps for multiple architectures, on multiple release channels from daily builds to stable releases.</p>
```

Comment:
Could you split these template changes out into their own PR? They don't seem related to the code restructure, and I don't right now understand what they're doing, so it would be easier if they were in their own PR where I could focus on what they're doing.